Update staking and rates documentation (#1269)
smartcontractkit/documentation: src/pages/data-feeds/rates-feeds/index.mdx

## ETH Staking APR

The ETH Staking APR feeds provides a trust-minimized and tamper-proof source of truth for the global rate of return from staking as a validator to secure the Ethereum network. The annualized rate of return is calculated over 30-day and 90-day rolling windows. Data providers use off-chain computation to calculate returns at an epoch level, reach consensus on the APR, and then write the results on-chain to be used by decentralized protocols and Web 3 applications. Feeds are currently configured to update at a minimum of once per day.

<ClickToZoom src="/images/data-feed/eth-staking-apr-feed-V3.png" />

See the [Rate and Volatility Feed Addresses](/data-feeds/rates-feeds/addresses) page to find the ETH Staking APR feeds that are currently available. If you have questions or would like to request an enhancement to ETH Staking APR feeds, contact us using [this form](https://forms.gle/Jc1MGY5ry1ceYH1X6).

## Realized volatility

> Realized volatility measures how much an asset price has moved around in the past. This value is expressed as a percent of the asset price. The more an asset price moves up or down over time, the higher the realized volatility is for that asset. Please note that realized volatility is not the same as implied volatility, which is a measure of the market’s expectation about future volatility typically derived from options markets.

Reviewer: ```suggestion
Realized volatility measures asset price movement over a specific time interval. This value is expressed as a percent of the asset price. The more an asset price moves up or down over time, the higher the realized volatility is for that asset. Please note that realized volatility is not the same as implied volatility, which measures the market’s expectation about future volatility typically derived from options markets.
```

> Each data feed reflects the volatility of an asset over a specific rolling window of time. For example, there are data feeds that provide volatility data for the last 24 hours, 7 days, and 30 days of time. You can compare the data across these windows to infer whether the volatility of an asset is trending up or down. For example, if realized volatility for the 24-hour window is higher than the 7-day window, this suggests volatility might be increasing.

Reviewer: ```suggestion
Each data feed reflects the volatility of an asset over a specific rolling window of time. For example, some data feeds provide volatility data for the last 24 hours, 7 days, and 30 days of time. You can compare the data across these windows to infer whether the volatility of an asset is trending up or down. For example, if realized volatility for the 24-hour window is higher than the 7-day window, volatility might increase.
```

> The same high-quality data providers used in Chailink’s price feeds sample price data every 10 minutes to refresh volatility estimates, and on-chain values are updated when the feed heartbeat or deviation threshold are met.

Reviewer: ```suggestion
The same high-quality data providers used in Chailink’s price feeds sample price data every 10 minutes to refresh volatility estimates. On-chain values are updated when the feed heartbeat or deviation threshold is met.
```
